ompi/mca/coll/coll.h

```diff
@@ -589,6 +621,12 @@ struct mca_coll_base_module_2_3_0_t {
     mca_coll_base_module_alltoallv_init_fn_t coll_neighbor_alltoallv_init;
     mca_coll_base_module_neighbor_alltoallw_init_fn_t coll_neighbor_alltoallw_init;
 
+    /* fault tolerant collective functions */
```

Comment:
Possible module ABI compat issue @bwbarrett

Reply:
Sorry I had to force push to the branch because there was a problem with the way I did the squash initially. The links in these comments do not work anymore.